### PR TITLE
update basic features for 7.1

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Update basic features for v7.1

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13299
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [Optimizer hints](/optimizer-hints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
-| [MPP 执行引擎](/explain-mpp.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
-| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) | Y | Y | N | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md) | Y | Y | Y | Y | Y | E | E | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) | Y | Y | Y | Y | E | E | N | N | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E | E |
+| 高级 SQL 功能 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y | Y | Y | E | E | E | E |
+| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:94]:
```suggestion
| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [更改列类型](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
-| [临时表](/temporary-tables.md) | Y | Y | Y | Y | Y | Y | N | N | N | N |
-| 并行 DDL | Y | Y | Y | N | N | N | N | N | N | N |
-| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) | Y | Y | Y | N | N | N | N | N | N | N |
-| [元数据锁](/metadata-lock.md) | Y | Y | Y | N | N | N | N | N | N | N |
-| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| 数据定义语言 (DDL) | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| `CREATE`，`DROP`，`ALTER`，`RENAME`，`TRUNCATE` | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [生成列](/generated-columns.md) | E | E | E | E | E | E | E | E | E |
```
Comment (ran-huang) [inline basic-features.md:113]:
```suggestion
| [生成列](/generated-columns.md) | Y | E | E | E | E | E | E | E | E |
```

File: basic-features.md [hunk 3]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [RocksDB 引擎](/storage-engine/rocksdb-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Titan 插件](/storage-engine/titan-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge实验功能) | E | E | E | E | E | E | E | E | E |
+| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E | E | E | N | N | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-add-index.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [复合主键](/constraints.md#主键约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [唯一约束](/constraints.md#唯一约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | N | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:61]:
```suggestion
| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`AUTO_RANDOM` 列](/auto-random.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TTL (Time to Live)](/time-to-live.md) | Y | E | N | N | N | N | N | N | N |
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| 在单条语句中添加多列 | Y | Y | E | E | E | E | E | E | E |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [临时表](/temporary-tables.md) | Y | Y | Y | Y | Y | N | N | N | N |
+| 并行 DDL | Y | Y | N | N | N | N | N | N | N |
+| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) | Y | Y | N | N | N | N | N | N | N |
+| [元数据锁](/metadata-lock.md) | Y | Y | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:126]:
```suggestion
| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | N | N | N | N | N | N | N |
| [暂停/恢复 DDL](https://docs.pingcap.com/zh/tidb/dev/ddl-introduction#ddl-相关的命令介绍) | E | N | N | N | N | N | N | N | N |
```

Comment (ran-huang) [reply basic-features.md:126]: @benmaoer 请看下 **暂停/恢复 DDL** 是否要加入基本特性列表

File: basic-features.md [hunk 5]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Titan Level Merge](/storage-engine/titan-configuration.md#level-merge实验功能) | E | E | E | E | E | E | E | E | E |
+| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E | E | E | N | N | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-add-index.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [复合主键](/constraints.md#主键约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [唯一约束](/constraints.md#唯一约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | N | N | N | N | N | N | N | N |
+| [外键约束](/constraints.md#外键约束) | Y | N | N | N | N | N | N | N | N |
+| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | E | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:63]:
```suggestion
| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 6]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [公共表表达式 (CTE)](/sql-statements/sql-statement-with.md) | Y | Y | Y | Y | Y | Y | Y | N | N |
+| `START TRANSACTION`，`COMMIT`，`ROLLBACK` | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [用户自定义变量](/user-defined-variables.md) | E | E | E | E | E | E | E | E | E |
+| [`BATCH [ON COLUMN] LIMIT INTEGER DELETE`](/sql-statements/sql-statement-batch.md) | Y | Y | Y | N | N | N | N | N | N |
+| [`BATCH [ON COLUMN] LIMIT INTEGER INSERT/UPDATE/REPLACE`](/sql-statements/sql-statement-batch.md) | Y | Y | N | N | N | N | N | N | N |
+| [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md) | Y | Y | E | N | N | N | N | N | N |
+| [表级锁 (Table Lock)](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | E | E | E | E | E | E | E | E | E |
+| [物化列式存储的查询结果](/tiflash/tiflash-results-materialization.md) | E | E | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:87]:
```suggestion
| [物化列式存储的查询结果](/tiflash/tiflash-results-materialization.md) | Y | E | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 7]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Range 分区](/partitioned-table.md#range-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Hash 分区](/partitioned-table.md#hash-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Key 分区](/partitioned-table.md#key-分区) | Y | N | N | N | N | N | N | N | N |
+| [List 分区](/partitioned-table.md#list-分区) | Y | Y | Y | E | E | E | E | E | N |
+| [List COLUMNS 分区](/partitioned-table.md#list-columns-分区) | Y | Y | Y | E | E | E | E | E | N |
+| [`EXCHANGE PARTITION`](/partitioned-table.md) | Y | Y | E | E | E | E | E | E | N |
+| [`REORGANIZE PARTITION`](/partitioned-table.md#重组分区) | Y | N | N | N | N | N | N | N | N |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | E | E | E | E | N | N |
+| [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) | Y | Y | N | N | N | N | N | N | N |
+| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | E | E | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:153]:
```suggestion
| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | Y | E | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 8]
```diff
@@ -20,218 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | E | E | E | N | N | N | N | N | N | N |
+| 分区 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Range 分区](/partitioned-table.md#range-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Hash 分区](/partitioned-table.md#hash-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Key 分区](/partitioned-table.md#key-分区) | Y | N | N | N | N | N | N | N | N |
+| [List 分区](/partitioned-table.md#list-分区) | Y | Y | Y | E | E | E | E | E | N |
+| [List COLUMNS 分区](/partitioned-table.md#list-columns-分区) | Y | Y | Y | E | E | E | E | E | N |
+| [`EXCHANGE PARTITION`](/partitioned-table.md) | Y | Y | E | E | E | E | E | E | N |
+| [`REORGANIZE PARTITION`](/partitioned-table.md#重组分区) | Y | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:150]:
```suggestion
| [`REORGANIZE PARTITION`](/partitioned-table.md#重组分区) | Y | N | N | N | N | N | N | N | N |
| [`COALESCE PARTITION`](/partitioned-table.md#减少分区数量) | Y | N | N | N | N | N | N | N | N |
```

**Review by joey-yez — APPROVED** [review]

File: basic-features.md (hunk 4 above)
Comment (benmaoer) [reply basic-features.md:126]: LGTM

**Review by benmaoer — APPROVED** [review]
LGTM

File: basic-features.md [hunk 9]
```diff
@@ -20,218 +20,219 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`SET CONFIG`](/dynamic-config.md) | Y | Y | Y | E | E | E | E | E | E |
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | N | N | N | N | N | N |
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | E | N | N | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | N | N | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | N | N | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | E | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:234]:
```suggestion
| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | N | N | N | N | N | N | N | N |
```
@hfxsd @songrijie PTAL

File: basic-features.md (hunk 1 above)
Comment (Oreoxmt) [reply basic-features.md:94]: @songrijie PTAL

File: basic-features.md (hunk 9 above)
Comment (hfxsd) [reply basic-features.md:235]: LGTM

File: basic-features.md [hunk 10]
```diff
@@ -241,6 +243,6 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 
 [^4]: 从 [TiDB v6.4.0](/releases/release-6.4.0.md) 开始，支持[高性能、全局单调递增的 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
 
-[^5]: 对于 [TiDB v7.0.0](/releases/release-7.0.0.md)，新增参数 `FORMAT`、`FIELDS DEFINED NULL BY`、`With batch_size=<number>,detached`，以及新增支持从 S3 和 GCS 导入数据，均为实验特性。
+[^5]: 对于 [TiDB v7.1.0](/releases/release-7.0.0.md)，新增的并行导入功能、集成 Physical Import Mode 功能，均为实验特性。
```
Comment (hfxsd) [inline basic-features.md:246]: @Frank945946 PTAL

File: basic-features.md [hunk 11]
```diff
@@ -20,218 +20,220 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
-| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage/GCP)](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | E | E | N | N | N | N | N | N | N |
-| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | Y | N | N | N | N | N | N | N |
-| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | N | N | N | N | N | N | N | N | N |
+| 数据导入和导出 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | E [^5] | Y | Y | Y | Y | Y | Y | Y | N [^6] |
```
Comment (hfxsd) [inline basic-features.md:197]: 这个是不是改成 Y 比较好？毕竟大部分 feature 都 GA 了。
```suggestion
| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y [^5] | Y | Y | Y | Y | Y | Y | Y | N [^6] |
```

File: basic-features.md (hunk 10 above)
Comment (Frank945946) [inline basic-features.md:246]:
```suggestion
[^5]: 对于 [TiDB v7.1.0](/releases/release-7.0.0.md)，新增的并发导入功能、集成 Physical Import Mode 功能，均为实验特性。
```

File: basic-features.md (hunk 1 above)
Comment (songrijie) [reply basic-features.md:94]: LGTM

File: basic-features.md (hunk 9 above)
Comment (songrijie) [reply basic-features.md:235]: LGTM

**Review by ran-huang — APPROVED** [review]

File: basic-features.md (hunk 11 above)
Comment (Oreoxmt) [reply basic-features.md:197]: @Frank945946 PTAL

File: basic-features.md [hunk 12]
```diff
@@ -241,6 +243,6 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 
 [^4]: 从 [TiDB v6.4.0](/releases/release-6.4.0.md) 开始，支持[高性能、全局单调递增的 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
 
-[^5]: 对于 [TiDB v7.0.0](/releases/release-7.0.0.md)，新增参数 `FORMAT`、`FIELDS DEFINED NULL BY`、`With batch_size=<number>,detached`，以及新增支持从 S3 和 GCS 导入数据，均为实验特性。
+[^5]: 对于 [TiDB v7.1.0](/releases/release-7.1.0.md)，新增的并发导入功能、集成 Physical Import Mode 功能，均为实验特性。
```
Comment (Oreoxmt) [inline basic-features.md:246]:
```suggestion
[^5]: 对于 [TiDB v7.1.0](/releases/release-7.1.0.md)，新增的并发导入和集成 Physical Import Mode 功能，均为实验特性。
```

**Review by songrijie — APPROVED** [review]

File: basic-features.md [hunk 13]
```diff
@@ -241,6 +243,6 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 
 [^4]: 从 [TiDB v6.4.0](/releases/release-6.4.0.md) 开始，支持[高性能、全局单调递增的 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
```
Comment (ran-huang) [inline basic-features.md:244]: 正文里没有出现过 [^4]，是否可以删除了？

**Review by ran-huang — APPROVED** [review]

File: basic-features.md [hunk 14]
```diff
@@ -20,218 +20,220 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) | Y | Y | Y | N | N | N | N | N | N | N |
-| [元数据锁](/metadata-lock.md) | Y | Y | Y | N | N | N | N | N | N | N |
-| [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| 数据定义语言 (DDL) | 7.1 (upcoming) | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| `CREATE`，`DROP`，`ALTER`，`RENAME`，`TRUNCATE` | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [生成列](/generated-columns.md) | Y | E | E | E | E | E | E | E | E |
+| [视图](/views.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [序列](/sql-statements/sql-statement-create-sequence.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`AUTO_INCREMENT` 列](/auto-increment.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (Oreoxmt) [inline basic-features.md:116]:
```suggestion
| [`AUTO_INCREMENT` 列](/auto-increment.md) | Y | Y[^4] | Y | Y | Y | Y | Y | Y | Y |
```

Comment (Oreoxmt) [reply basic-features.md:116]: @ran-huang [^4] 加到了 6.5 这一列

**Review by qiancai — APPROVED** [review]

**Pull request merged**